tools/WinObjC.Packaging/WinObjC.Packager.targets

```diff
@@ -0,0 +1,79 @@
+    <MSBuild Projects="$(SolutionPath)"
+       Targets="GetSolutionConfigurationContents"
+       Properties="Configuration=$(Configuration);Platform=$(Platform)">
+      <Output TaskParameter="TargetOutputs" ItemName="_SolutionConfigurationContents" />
+    </MSBuild>
+
+    <Message Text="AddSingleConfigBuildOutput Platform: $(Platform)" Importance="High"/>
+
+    <MSBuild Projects="$(MSBuildProjectFile)"
+       Targets="GetPackageContents"
+       Properties="SkipGetPackageContentsForOtherPlatforms=true;BuildingInsideVisualStudio=false;BuildProjectReferences=true;IsPackable=false;Configuration=$(Configuration);Platform=$(Platform);CurrentSolutionConfigurationContents=%(_SolutionConfigurationContents.Identity);GetPackageContentsDependsOn=Build;$(GetPackageContentsDependsOn)">
+      <Output TaskParameter="TargetOutputs" ItemName="_PackageContentsForPlat" />
+    </MSBuild>
+  </Target>
+
+  <Target Name="AddOtherConfigsBuiltOutput" BeforeTargets="GetPackageContents" Condition="'$(SkipGetPackageContentsForOtherPlatforms)' != 'true' AND '$(Platform)' == 'AnyCPU'">
+    <ItemGroup>
+      <_PackTargetPlat Include="%(ProjectConfiguration.Platform)" Condition="'%(ProjectConfiguration.Configuration)' == '$(Configuration)' AND '%(ProjectConfiguration.Platform)' != '$(Platform)'" />
+    </ItemGroup>
+
+    <MSBuild Projects="$(MSBuildProjectFile)"
+       Targets="AddSingleConfigBuildOutput"
+       Properties="Platform=%(_PackTargetPlat.Identity)">
+      <Output TaskParameter="TargetOutputs" ItemName="_OtherPackageContents" />
+    </MSBuild>
+
+    <ItemGroup>
+      <PackageFile Include="@(_OtherPackageContents)" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="AddBuiltOutput" BeforeTargets="GetPackageContents" Condition="'$(SkipGetPackageContentsForOtherPlatforms)' != 'true' and '$(PresetPackageContents)' == 'true'">
+    <ItemGroup>
+      <PackageFile Include="$(MSBuildThisFileDirectory)Package.props" Condition="Exists('$(MSBuildThisFileDirectory)Package.props')">
+        <PackagePath>build\$(PackageId).props</PackagePath>
+      </PackageFile>
+      <PackageFile Include="$(MSBuildThisFileDirectory)Package.targets" Condition="Exists('$(MSBuildThisFileDirectory)Package.targets')">
+        <PackagePath>build\$(PackageId).targets</PackagePath>
+      </PackageFile>
+      <PackageFile Include="$(MSBuildThisFileDirectory)Package.Default.props" Condition="Exists('$(MSBuildThisFileDirectory)Package.Default.props')">
+        <PackagePath>build\$(PackageId).Default.props</PackagePath>
+      </PackageFile>
+      <PackageFile Include="$(MSBuildThisFileDirectory)Package.Default.targets" Condition="Exists('$(MSBuildThisFileDirectory)Package.Default.targets')">
+        <PackagePath>build\$(PackageId).Default.targets</PackagePath>
+      </PackageFile>
+      <PackageFile Include="$(MSBuildProjectDirectory)\$(PackageId).Custom.targets" Condition="Exists('$(MSBuildProjectDirectory)\$(PackageId).Custom.targets')">
```

Comment:
>MSBuildProjectDirectory [](start = 105, length = 23)

can you put a comment for the custom ones about them living in the project folder. Just scanning this its hard to see this one is special if someone comes along later and fixes something up